### ISSUE TITLE
EnableRelaxedMemberAccess does not work on ScriptFunctionCall

### ISSUE THREAD

**Issue by NicMay** [lunet-io/scriban#200]
Currently an Exception is thrown, if the script function is not found
In https://github.com/lunet-io/scriban/tree/master/src/Scriban/Syntax/ScriptFunctionCall.cs
The current code is:
```
public override object Evaluate(TemplateContext context)
        {
            // Invoke evaluate on the target, but don't automatically call the function as if it was a parameterless call.
            var targetFunction = context.Evaluate(Target, true);

            // Throw an exception if the target function is null
            if (targetFunction == null)
            {
                throw new ScriptRuntimeException(Target.Span, $"The function `{Target}` was not found");
            }

            return Call(context, this, targetFunction, context.AllowPipeArguments, Arguments);
        }
```

Should it be more like the following?

```
public override object Evaluate(TemplateContext context)
        {
            // Invoke evaluate on the target, but don't automatically call the function as if it was a parameterless call.
            var targetFunction = context.Evaluate(Target, true);

            // Throw an exception if the target function is null
            if (targetFunction == null)
            {
               if ( !context.EnableRelaxedMemberAccess)
                      throw new ScriptRuntimeException(Target.Span, $"The function `{Target}` was not found");
                else
                       return string.Empy;
            }

            return Call(context, this, targetFunction, context.AllowPipeArguments, Arguments);
        }
```

**Comment by xoofx** [comment]
Yep, it's an oversight, PR welcome

**Comment by xoofx** [comment]
Fixed in next 2.1.3 later today
